ludwig/config_validation/checks.py

```diff
@@ -322,6 +322,21 @@ def check_tagger_decoder_requirements(config: "ModelConfig") -> None:  # noqa: F
         )
 
 
+@register_config_check
```

Comment:
Discussed offline. No longer needed if `FILL_WITH_CONST` is removed as an option.